templates/web/common/includes/display_blocks.tt.html

```diff
@@ -0,0 +1,19 @@
+<!-- start templates/[% template.name %] -->
+
+[% FOREACH block IN blocks %]
+
+    <div class="block">
+        <h3 class="block_title">[% block.block_ref_title %]</h3>
```

Comment:
Here you can check if [% block.block_ref_title %] is empty, if it is, don't display the <h3> block.